generics/services/survey.js

```diff
@@ -5,757 +5,873 @@
  * Description : Assessment service related information.
  */
 
- //dependencies
-const request = require('request');
+//dependencies
+const request = require('request')
 
-const ASSESSMENT_URL = process.env.ML_SURVEY_SERVICE_URL;
+const SURVEY_SERVICE_URL = process.env.SURVEY_SERVICE_URL
+/**
+ * Assessment create
+ * @function
+ * @name createAssessmentSolutionFromTemplate
+ * @param {String} token - logged in user token.
+ * @param {String} templateId - template id.
+ * @param {Object} bodyData - Body data
+ * @returns {JSON} - Create assessment from template solution.
+ */
+
+const createAssessmentSolutionFromTemplate = function (token, templateId, bodyData) {
```

Comment:
@praveenKDass how is this different from observation? If not used now just keep observation and survey